assignments/4. operators/conditional-operator.md

```diff
@@ -59,4 +135,32 @@ Take the value of `marks` (0-100) from user using `prompt` and `alert` the messa
 * [ ] `FF` if `marks` is less than or equal to 30
 ```js
 // Your code goes here
-```
+
+let marks = +prompt("Enter your marks");
+switch(marks) {
+    case(marks<=30):
+    alert("FF");
+    break;
+    case(marks<=40):
+    alert("DD");
+    break;
+    case(marks<=50):
+    alert("CD");
+    break;    
+    case(marks<=60):
+    alert("CC");
+    break;
+    case(marks<=70):
+    alert("BC");
+    break;
+    case(marks<=80):
+    alert("BB");
+    break;
+    case(marks<=90):
+    alert("AB");
+    break;
+    case(marks>=90):
+    alert("AA");
+    break;
+}
+```
```

Comment:
If you are switching over `marks` it will not work.
Say for example if the value is 64 you are looking for the case with the exact value of 64. 
But all the cases are using will return `true` or `false` so switch over `true`.